actors/builtin/miner/miner_actor.go

```diff
@@ -1541,18 +1542,39 @@ func terminateSectors(rt Runtime, sectorNos *abi.BitField, terminationType power
 			rt.Abortf(exitcode.ErrIllegalState, "failed to expand faults")
 		}
 
+		var skippedSectorNos []uint64 // sectors we're not terminating because they either don't exist or have expired.
```

Comment:
Perhaps "missing"? Just because "skipped" has a specific meaning wrt faults.

Reply:
`notTerminated`? It's not quite "missing" because we also skip non-expired sectors.